torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -313,6 +313,13 @@ class FullyShardedDataParallel(nn.Module, _FSDPState):
             prevent too many in-flight all-gathers. This ``bool`` only affects
             the sharded strategies that schedule all-gathers. Enabling this can
             help lower the number of CUDA malloc retries.
+        ignored_parameters (Optional[Iterable[torch.nn.Parameter]]): Ignored
+            parameters will not be managed by this FSDP instance,
+            that means these parameters will not be flattened and sharded by FSDP,
+            their gradients will not be synchronized as well. With this newly added
+            argument, `ignored_modules` could be deprecated soon. For back compatiablity,
```

Comment:
nit: Double backtick for code type in public docstrings (after the type annotation)
```suggestion
            argument, ``ignored_modules`` could be deprecated soon. For backward compatibility,
```

Single backtick only shows as italics:
![Screen Shot 2023-01-04 at 4 32 45 AM](https://user-images.githubusercontent.com/31054793/210525293-576cb05d-357b-4404-9fe3-879b8fe9a905.png)
Example with double backtick (`False`, `True`, `bool`):
![Screen Shot 2023-01-04 at 4 34 00 AM](https://user-images.githubusercontent.com/31054793/210525578-6e937753-d6c0-41d7-a3ef-a81836dc3cc8.png)